clusters/resource_cluster.go

```diff
@@ -383,6 +383,9 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, c *commo
 	if createClusterRequest.Autoscale == nil {
 		createClusterRequest.ForceSendFields = []string{"NumWorkers"}
 	}
+	if createClusterRequest.GcpAttributes != nil {
+		createClusterRequest.GcpAttributes.ForceSendFields = []string{"LocalSsdCount"}
```

Comment:
We should call GetOkExists for local_ssd_count to check that it was explicitly set and only then send it. The reason is that if not specified in request, the default value for local_ssd_count on backend is not 0 (can be 1, 2, 4 depending on type of instance used).